k8scrdclient/k8scrdclient.go

```diff
@@ -91,20 +91,34 @@ func (c *CRDClient) ensureCreated(ctx context.Context, crd *apiextensionsv1beta1
 			return microerror.Mask(err)
 		}
 
-		for _, cond := range manifest.Status.Conditions {
```

Comment:
The important change here is that we cannot simply iterate over the conditions once as there are different severities to the conditions. Further we have to check for "fatal" conditions first before we can "accept" the establishment of a given CRD.